common/src/main/java/com/datatorrent/common/util/PubSubWebSocketClient.java

```diff
@@ -376,7 +385,8 @@ public void unsubscribe(String topic) throws IOException
    * @return subscribe num subscriber message.
    * @throws IOException
    */
-  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws IOException
+  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws
+      IOException
```

Comment:
Consistency and readability.  You have other places that break before the throws, and it is odd to break the line after the throws keyword.  Imagine something like:
{code}
public class SomeLongClassName extends
    SomeOtherLongClassName implements
    SomeLongInterfaceNameA
{code}
versus
{code}
public class SomeLongClassName 
    extends SomeOtherLongClassName 
    implements SomeLongInterfaceNameA 
{code}
In the first case, SomeOtherLongClassName has nothing to do with the implements keyword, and yet they are on the same line.